sdk/src/main/res/layout/activity_paymaya_payment.xml

```diff
@@ -1,49 +1,55 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Please also revert changes to this file. PR should be okay to merge afterwards. Thanks.